src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
```

Comment:
the author of the message is defined in 3 places : 

* user_id
* recipients : `{type: "from", label: "my display name", protocol: "email", contact_id: "aienraseat", address: "me@mydomain.net"}, {type: "reply-to", …`
* identities: `{display_name: "my display name", identifier: "me@mydomain.net", type: "email_address", user_id: "anierstn"}`

In the UI, the user should choose a "Local Identity" to send the message from. This choice creates a local_identity  and add a recipient with type "from" within the draft.